streams/src/main/java/org/apache/kafka/streams/kstream/internals/KGroupedTableImpl.java

```diff
@@ -130,12 +134,27 @@ public KGroupedTableImpl(KStreamBuilder topology,
         Objects.requireNonNull(adder, "adder can't be null");
```

Comment:
Can remove the first two null checks as they will be covered by the overloaded `reduce`